### ISSUE TITLE
String object after render

### ISSUE THREAD

**Issue by DragorWW** [twigjs/twig.js#415]
if use autoescape: true, method render of template return String object. (Twig.Template.prototype.render)
if use autoescape: false , method render of template return string literal.

problem in final render:

``` javascript
        Twig.Markup = function(content, strategy) {
            if(typeof strategy == 'undefined') {
                strategy = true;
            }

            if (typeof content === 'string' && content.length > 0) {
                content = new String(content); // <<<<---- it return after template.render() if autoescape: true
                content.twig_markup = strategy;
            }
            return content;
        };
```

**Comment by ollien** [comment]
This was fixed with #577 and should be closed.

**Comment by RobLoach** [comment]
Thanks for the heads up, Ollien.
